GoogleMapsComponents/Maps/DirectionsRenderer.cs

```diff
@@ -34,9 +34,18 @@ public async Task<DirectionsResult> Route(DirectionsRequest request)
             var response = await _jsObjectRef.InvokeAsync<string>(
                     "googleMapDirectionServiceFunctions.route",
                     request);
-            var DirResult = JsonConvert.DeserializeObject<DirectionsResult>(response);
-
-            return DirResult;
+            try
+            {
+                var DirResult = JsonConvert.DeserializeObject<DirectionsResult>(response);
+
+                return DirResult;
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine("Error parsing DirectionsResult Object. Message: " + e.Message);
+                return null;
```

Comment:
If it is parsing exception why not use JsonSerializationException. Again i could be wrong. Also i think it is better print all stack. These error are not for use but for developer and they would like to know more

Reply:
This will actually create a JsonReaderException as well... so, I wanted to make sure that catch anything that comes in as an exception.